corehq/apps/es/es_query.py

```diff
@@ -418,14 +418,15 @@ def sort(self, field, desc=False, reset_sort=True):
         }
         return self._sort(sort_field, reset_sort)
 
-    def nested_sort(self, path, field_name, nested_filter, desc=False, reset_sort=True):
+    def nested_sort(self, path, field_name, nested_filter, desc=False, reset_sort=True, sort_missing=None):
         """Order results by the value of a nested field
         """
         sort_field = {
             '{}.{}'.format(path, field_name): {
                 'order': 'desc' if desc else 'asc',
                 'nested_path': path,
                 'nested_filter': nested_filter,
+                'missing': sort_missing,
```

Comment:
As far as I can tell ES ignores if there's a `None`/`null` value here, so I don't foresee any issues except having to update some tests.